onnxruntime/test/testdata/kernel_def_hashes/onnx.cpu.json

```diff
@@ -329,11 +329,11 @@
     ],
     [
         "Clip ai.onnx CPUExecutionProvider",
-        15464277967303611680
+        3243908767975096392
```

Comment:
@[edgchen1](https://github.com/edgchen1) do we need to update this hash , or add a new entry?

Reply:
@edgchen1 gentle ping

Reply:
the hash value should not change. you can use KernelDefBuilder::FixedTypeConstraintForHash()
more info here:
https://github.com/microsoft/onnxruntime/blob/dce5d719c5007bc56b5ecd2ba0fe8daf44771f16/onnxruntime/test/providers/kernel_def_hash_test.cc#L14-L31

Reply:
reverted back